composer.json

```diff
@@ -34,8 +34,6 @@
     },
     "require": {
         "php": "~8.1.0 || ~8.2.0 || ~8.3.0 || ~8.4.0",
-        "ext-dom": "*",
-        "ext-filter": "*",
         "ext-json": "*",
```

Comment:
should ext-json be kept since it is built in for a long time now and have to be explicitly disabled during compilation to be unavailable?